paperoni/sources/scrapers/openreview.py

```diff
@@ -0,0 +1,228 @@
+from datetime import datetime
+from coleo import Option, tooled
+import openreview
+
+from ..model import (
+    Author,
+    DatePrecision,
+    Link,
+    Paper,
+    Release,
+    Topic,
+    Venue,
+    VenueType
+)
+from ...utils import QueryError
+
+class OpenReviewScraper:
+    @tooled
+    def query(
+        self,
+        # Author to query
+        # [alias: -a]
+        # [nargs: +]
+        author: Option = [],
+        # Title of the paper
+        # [alias: -t]
+        # [nargs: +]
+        title: Option = [],
+        # Maximal number of results per query
+        block_size: Option & int = 100,
+        # Maximal number of results to return
+        limit: Option & int = 10000,
+    ):
+        title = " ".join(title)
+
+        if author and title:
+            raise QueryError("Cannot query both author and title")
+
+        client = openreview.Client(baseurl='https://api.openreview.net')
+        params = {
+            "content": {"venueid": 'NeurIPS.cc/2021/Conference'},
```

Comment:
Fill with `--venue` parameter.

Reply:
That was debug line but you do see a value in filtering by venue?